src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,90 @@
-export const Pagination = () => {};
+import React from 'react';
+import classNames from 'classnames';
+import { getPages, getArrOfItemsForCurrPage } from '../../utils';
+
+type Props = {
+  total: number
+  perPage: number
+  currentPage: number
+  onPageChange: (event: React.MouseEvent<HTMLAnchorElement>) => void
+};
+
+export const Pagination: React.FC<Props> = React.memo(
+  ({
+    total,
+    perPage,
+    currentPage,
+    onPageChange,
+  }) => {
+    const pages = getPages(total, perPage);
+
+    const arrOfItems = getArrOfItemsForCurrPage(42, perPage, currentPage);
```

Comment:
The first argument of `getArrOfItemsForCurrPage` should be `total` instead of a hardcoded value `42`.